src/class_addressGenerator.py

```diff
@@ -266,8 +264,8 @@ def run(self):
                             potentialPubSigningKey + potentialPubEncryptionKey)
                         ripe = RIPEMD160Hash(sha.digest()).digest()
                         if (
-                            ripe[:numberOfNullBytesDemandedOnFrontOfRipeHash] ==
-                            '\x00' * numberOfNullBytesDemandedOnFrontOfRipeHash
```

Comment:
make this into function, `check_ripe_leading_null`

Reply:
Will update in next PR